grouper/ctl/user.py

```diff
@@ -4,7 +4,8 @@
 from grouper.ctl.util import ensure_valid_username, make_session
 from grouper.model_soup import User
```

Comment:
is there a reason we can't renaming these existing imports to directly import the new model?